webdriver/tests/bidi/browsing_context/locate_nodes/invalid.py

```diff
@@ -0,0 +1,171 @@
+
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.browsing_context.locate_nodes(
+            context=value, locator={ "type": "css", "value": "div" }
+        )
+
+
+async def test_params_context_invalid_value(bidi_session, inline, top_context):
+    url = inline("""<div>foo<span><strong>bar</strong></span><span>BAR</span>baz</div>""")
+    await bidi_session.browsing_context.navigate(
+        context=top_context["context"], url=url, wait="complete"
+    )
+
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.browsing_context.locate_nodes(
+            context="foo", locator={ "type": "css", "value": "div" }
+        )
+
+
+@pytest.mark.parametrize("value", [False, 42, {}, []])
+async def test_params_locator_invalid_type(bidi_session, inline, top_context, value):
+    url = inline("""<div>foo<span><strong>bar</strong></span><span>BAR</span>baz</div>""")
+    await bidi_session.browsing_context.navigate(
+        context=top_context["context"], url=url, wait="complete"
+    )
+
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.browsing_context.locate_nodes(
+            context=top_context["context"], locator={ "type": value, "value": "div" }
+        )
+
+
+async def test_params_locator_invalid_value(bidi_session, inline, top_context):
+    url = inline("""<div>foo<span><strong>bar</strong></span><span>BAR</span>baz</div>""")
+    await bidi_session.browsing_context.navigate(
+        context=top_context["context"], url=url, wait="complete"
+    )
+
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.browsing_context.locate_nodes(
+            context=top_context["context"], locator={ "type": "invalid", "value": "div" }
+        )
+
+
+@pytest.mark.parametrize("value", [False, "string", {}, []])
+async def test_params_max_node_count_invalid_type(bidi_session, inline, top_context, value):
+    url = inline("""<div>foo<span><strong>bar</strong></span><span>BAR</span>baz</div>""")
+    await bidi_session.browsing_context.navigate(
+        context=top_context["context"], url=url, wait="complete"
+    )
+
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.browsing_context.locate_nodes(
+            context=top_context["context"],
+            locator={ "type": "css", "value": "div" },
+            max_node_count=value
+        )
+
+
+async def test_params_max_node_count_invalid_value(bidi_session, inline, top_context):
+    url = inline("""<div>foo<span><strong>bar</strong></span><span>BAR</span>baz</div>""")
+    await bidi_session.browsing_context.navigate(
+        context=top_context["context"], url=url, wait="complete"
+    )
+
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.browsing_context.locate_nodes(
+            context=top_context["context"],
+            locator={ "type": "invalid", "value": "div" },
+            max_node_count=0
```

Comment:
Lets also parameterize and check with `MAX_SAFE_INTEGER + 1`.

Reply:
Done in subsequent commit.